src/index.css

```diff
@@ -1,68 +1,14 @@
-:root {
-  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
-  line-height: 1.5;
-  font-weight: 400;
-
-  color-scheme: light dark;
-  color: rgba(255, 255, 255, 0.87);
-  background-color: #242424;
-
-  font-synthesis: none;
-  text-rendering: optimizeLegibility;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-a {
-  font-weight: 500;
-  color: #646cff;
-  text-decoration: inherit;
-}
-a:hover {
-  color: #535bf2;
-}
+@import url(./vendor/normalize.css);
+@import url(./vendor/fonts.css);
 
+html,
 body {
+  height: fit-content;
   margin: 0;
```

Comment:
Selectors by `tag` are **forbidden** by BEM methodology and generally bad practice, as it makes the CSS rules more specific. Please, use **BEM entities** instead of tag selectors.


Only `normalize.css` can have such tag selectors

Please, add a `body` class name for the `body` if you want to style it. You don't need the `html`  tag styling if you style `body`